tests/zfs-tests/include/libtest.shlib

```diff
@@ -119,6 +119,21 @@ function is_freebsd
 	fi
 }
 
+# Determine if this is a DilOS test system
+#
+# Return 1 if platform DilOS, 0 if otherwise
+
+function is_dilos
+{
+	typeset id=""
+	[[ -f /etc/os-release ]] && . /etc/os-release
+	if [[ $id == "dilos" ]]; then
+		return 1
```

Comment:
This should be the opposite, return 0 if it is Dilos, 1 if not.

Reply:
fixed, thanks